investigate unit test network call

> FAILED tests/signing/sigstore_test.py::TestSigstoreSigning::test_sign_identity_token_precedence - sigstore.errors.TUFError: Failed to refresh TUF metadata
> ```
> 
> seems like something we should fix in terms of unit tests relying out to network

Yeah, probably I missed something during mocking.

_Originally posted by @mihaimaruseac in https://github.com/sigstore/model-transparency/issues/400#issuecomment-2770828250_

--- a/tests/_signing/sigstore_test.py
+++ b/tests/_signing/sigstore_test.py
@@ -276,7 +276,9 @@ class TestSigning:
         expected_manifest = self._verify_dsse_signature(signature_path)
         assert expected_manifest == manifest
 
-    def test_sign_identity_token_precedence(self, mocked_oidc_provider):
+    def test_sign_identity_token_precedence(
+        self, mocked_oidc_provider, mocked_sigstore_signer
+    ):
         signer = sigstore.Signer(identity_token="provided_token")
         token = signer._get_identity_token()
         assert token == "provided_token"

